src/main/less/editor.less

```diff
@@ -290,3 +290,104 @@
     font-size:16px;
     font-family: @font-family-header;
 }
+
+//--[ Step Select Dialog ]----------------------------------------------------------------------------------------------------
+
+.placeholder(@rules) {
+    &::-webkit-input-placeholder {
+        @rules();
+    }
+
+    &:-moz-placeholder { /* Firefox 18- */
+        @rules();
+    }
+
+    &::-moz-placeholder {  /* Firefox 19+ */
+        @rules();
+    }
+
+    &:-ms-input-placeholder {  
+        @rules();
+    }
+}
+
+.editor-step-selection-dialog {
+    margin: -32px -32px -16px -32px;
```

Comment:
@sophistifunk Well, there's always padding in the JDL version, but the mockup has the search box spanning the entire dialog, so somewhere that padding needs to get removed... is there an option to alter the container styles via properties (I didn't see any when I looked)?